### DISCUSSION TITLE
.GLB files not accessible on certain FTP servers

### DISCUSSION THREAD

**Discussion by QNorbert** [google/model-viewer#3413] [Q&A]
Hi there,

For some reason the models won't open on Android and we get an can't load message. The strange thing is however, when hosting the files on a different server it does work. This makes me think some of our server settings are wrong. Both of the servers have SSL enabled and we are 100% positive the files are working correctly since the other server gives us no issues. Does anyone have any idea what might go wrong?

Kind regards,

![screenshot_20220503-101933_google](https://user-images.githubusercontent.com/77777801/166423602-890b60af-8627-4d0b-bc35-9da98179390e.jpg)

**Comment by elalish** [comment]
This looks like Scene Viewer; that's outside the browser, so if you're using cookies or some kind of token for authentication, that won't work outside of the browser. I'd recommend our default WebXR mode in this case, since it stays in the browser (it's also faster since it doesn't have to redownload the model).
